src/predicates/isconvex.jl

```diff
@@ -60,7 +60,10 @@ isconvex(::Tetrahedron) = true
 
 isconvex(p::Polygon{2}) = Set(vertices(convexhull(p))) == Set(vertices(p))
 
-isconvex(m::Multi{Dim,T}) where {Dim,T} = isapprox(measure(convexhull(m)), measure(m), atol=atol(T))
+function isconvex(m::Multi{Dim}) where {Dim}
+  x = measure(convexhull(m))
+  isapprox(x, measure(m), atol=atol(x))
```

Comment:
define two auxiliary variables to retain symmetry, these are two measures that will be compared with isapprox.